### ISSUE TITLE
'Take a picture using the camera' page issue

### ISSUE THREAD

**Issue by stephenhuey** [flutter/website#3907]
Page URL: https://flutter.dev/docs/cookbook/plugins/picture-using-camera.html
Page source: https://github.com/flutter/website/tree/master/src/docs/cookbook/plugins/picture-using-camera.md

Found a typo? You can fix it yourself by going to the page source and clicking the pencil icon. Or finish creating this issue.

Description of issue:

Tried to deploy this example to iOS 13.3.1 but I'm seeing this error: 

Failed to send request: {"jsonrpc":"2.0","id":"87","method":"getStack","params":{"isolateId":"isolates/3518422606420823"}}
Failed to send request: {"jsonrpc":"2.0","id":"86","method":"getMemoryUsage","params":{"isolateId":"isolates/3518422606420823"}}
Failed to send request: {"jsonrpc":"2.0","id":"88","method":"getObject","params":{"isolateId":"isolates/3518422606420823","objectId":"classes/3876"}}

Flutter 1.12.13+hotfix.8 with Android Studio 3.6 and Xcode 11.3.1 on Mac OS X Mojave 10.14.6 (so my tools are recent and I'm using the latest package dependencies mentioned in the tutorial.

**Comment by sfshaza2** [comment]
@johnpryan, can you take a look?

**Comment by johnpryan** [comment]
I think this is the same issue as https://github.com/flutter/flutter/issues/36150 - @stephenhuey can you check that you are using the latest version of the Flutter plugin for Android Studio?

**Comment by stephenhuey** [comment]
Looks like my Flutter plugin in Android Studio is on 44.0.2 and 45.1.1 is available - you think that would make a difference?  Just asking since I have some other work in flight and am being cautious at the moment.

**Comment by johnpryan** [comment]
I'm not sure if it would, 44.0.2 is a fairly recent version (the issue I linked is much older than your tools) - could you file an issue in flutter/flutter instead and link it here?

**Comment by sfshaza2** [comment]
Or, I could move this issue to the flutter/flutter repo. Just let me know!

**Comment by johnpryan** [comment]
Yeah, let's move it there, this doesn't seem to be related to the cookbook content.
